src/Microsoft.VisualStudio.Threading/TplExtensions.cs

```diff
@@ -23,13 +23,14 @@ public static partial class TplExtensions
         /// A singleton completed task.
         /// </summary>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Security", "CA2104:DoNotDeclareReadOnlyMutableReferenceTypes")]
+        [Obsolete("Use Task.CompletedTask instead.")]
         public static readonly Task CompletedTask = Task.FromResult(default(EmptyStruct));
 
         /// <summary>
         /// A task that is already canceled.
         /// </summary>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Security", "CA2104:DoNotDeclareReadOnlyMutableReferenceTypes")]
-        public static readonly Task CanceledTask = ThreadingTools.TaskFromCanceled(new CancellationToken(canceled: true));
+        public static readonly Task CanceledTask = Task.FromCanceled(new CancellationToken(canceled: true));
```

Comment:
💭 I wouldn't complain if these two were also obsolete, but they do still work fine

Reply:
Ya, I'm divided on that, so thanks for bringing it up. My API is more convenient since you don't have to supply a CancellationToken, and I bet most folks who call this API today don't have one. So I didn't want to force them all to come up with one. But I'm open.
In fact [index](http://index/?leftProject=Microsoft.VisualStudio.Threading&leftSymbol=fyyg7adqul4k) suggests we don't even have any users of this (at least not within that db). GitHub and Azure Repos show 0 and 1 result respectively as well. I guess I'll go ahead and deprecate it too.